cowprotocol/cowswap
# [ETH-FLOW-V2] Adjust price quotes for Native currency

# Summary

Deals with native currency quotes needing to have additional props specified in the payload object:
```ts
const ETH_FLOW_AUX_QUOTE_PARAMS = {
  signingScheme: 'eip1271',
  onchainOrder: true,
}
```

Adds an `isEthFlow` check via hook to places where we calculate quotes

## QA
1. test the app for normal orders (non-eth-flow) to make sure quotes/orders go thru
2. open `Network` tab in debugger
3. filter by `quote` 
4. select NATIVE <> ERC20
5. check that at least 2 quote requests where ETH is the sell token has:
```
signingScheme: 'eip1271',
onchainOrder: true
```
